2021/01/13 09:13:18 : Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 665, in storage_data
    getattr(self.get_source_cart(), 'finish_storage_data')()
  File "/home/hoang/Desktop/v32/cartmigration/models/basecart.py", line 3929, in finish_storage_data
    folder_upload = self.get_folder_upload(self._migration_id)
  File "/home/hoang/Desktop/v32/cartmigration/models/basecart.py", line 3941, in get_folder_upload
    folder_upload = root_path + '/' + DIR_UPLOAD + '/' + get_config_ini('src', 'file', file = 'test.ini')
TypeError: must be str, not NoneType

2021/01/13 09:13:32 : customers_errors 1: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 3787, in customer_import
    'user_nicename': convert['user_name'],
KeyError: 'user_name'

2021/01/13 09:13:32 : customers_errors 3: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 3787, in customer_import
    'user_nicename': convert['user_name'],
KeyError: 'user_name'

